inc/common/purge.php

```diff
@@ -19,7 +19,7 @@
 add_action( 'delete_link'                                           , 'rocket_clean_domain' );  // When a link is deleted.
 add_action( 'customize_save'                                        , 'rocket_clean_domain' );  // When customizer is saved.
 add_action( 'update_option_theme_mods_' . get_option( 'stylesheet' ), 'rocket_clean_domain' ); // When location of a menu is updated.
-
+add_action( 'upgrader_process_complete'                             , 'rocket_clean_domain' );  // When a theme or plugin is updated.
```

Comment:
`rocket_clean_domain` can't be used as the direct callback for this, as the parameters of the hook and the function don't match.

Also we don't want to auto clean for plugins update, so we need to check in the callback function that the  type of update is `theme` before triggering the clean.